### DISCUSSION TITLE
How do we do reporting or export all assets into csv format?

### DISCUSSION THREAD

**Discussion by chwijaya** [Shelf-nu/shelf.nu#1282] [Q&A]
Hi,

How do we export all assets into a csv file?
Is there any reports functionality?

thanks.

**Comment by DonKoko** [comment]
hello, @chwijaya.

Currently we only have a backup export option that exports a csv with some raw json of the assets. You can do that in the workspace settings.
As for reports, we have a lot of requests for it so we are currently in the process of defining and designing a module in our app to help users with reporting.
